### DISCUSSION TITLE
An admin user can add/manage other users (per-seat pricing)

### DISCUSSION THREAD

**Discussion by con-dog** [Glench/ExtPay#163] [Q&A]
Is it possible to have an admin user that can add and manage other users? 

I have a B2B extension, and I want a single admin to be able to add other emails , and to pay per user added.

They should be able to add/remove users.

Their bill should change based on the number of users

Is this currently possible with extension pay?

**Comment by burkybang** [comment]
It's not possible right now, but some of my customers have inquired about this as well. This would be so helpful.

**Reply by con-dog** [reply]
How are you handling customers that want this? Are you getting them to email you and then you handle it manually?

**Reply by burkybang** [reply]
I don’t know how to handle it manually. I'm probably just losing potential customers.

**Reply by con-dog** [reply]
Whats your extension? That sucks man. I'm investigating Firebase Cloud Functions + Stripe for my own extension because I need per seat payment and admin

**Reply by burkybang** [reply]
https://burkybang.com/SuiteAdvanced

**Comment by JPaulHendricks** [comment]
I would def use this. Thanks!

**Comment by Divya6696** [comment]
Just came to ask if this was possible. Would also like the developer email to have admin privileges for the free trail dashboard. I know you can currently add free users, but it would be helpful if removing users was also possible.
